src/endpoint/s3/ops/s3_put_bucket_notification.js

```diff
@@ -20,6 +24,23 @@ async function put_bucket_notification(req) {
         }
     }
 
+    //test new notifications.
+    //if test notification fails, fail the put op
+    let config_fs = null;
+    if (req.object_sdk.nsfs_config_root) {
+        config_fs = new ConfigFS(req.object_sdk.nsfs_config_root);
```

Comment:
I'm not sure adding `ConfigFS` as a dependency in an S3 op is something that we would like.
It might create coupling between the deployment way, and I would like to avoid it - the implementation according to the deployment (NC / Containerized) is in inner layers and not here.
Could you think of another alternative?

Reply:
Not exactly, but once/if we merge https://github.com/noobaa/noobaa-core/pull/8684, then I will only need req.object_sdk.nsfs_config_root because connection file directory will not be any directory specified in the config file, but rather only under the config root directory.

Reply:
Can it be moved to the `notification_utils`? (where I see that you already have modules related to FS.